### DISCUSSION TITLE
My Faculty Application was verified -- but my organizations aren't

### DISCUSSION THREAD

**Discussion by BillNace** [community/community#52331] [GitHub Education]
Greetings,

  I recently went through the academic reverification process and my application was verified. Yay!

  In the email from GitHub Education which requested the reverification, this paragraph appears:
<img width="621" alt="image" src="https://user-images.githubusercontent.com/1286667/230734683-3410d372-9596-4e18-b064-311f1b21c0f5.png">

  I returned to my benefits page.  But, I see nothing about my other organizations being upgraded.  How does GitHub Education detect my organizations? They are listed in my Github profile and I have the same billing email address on those orgs as my own.

  Any help?

Cheers,
Bill Nace

**Comment by samus-aran** [comment]
Hi @BillNace 👋 

Did you have your organisations upgraded to start off with or is this new?

You should be able to upgrade your organisations via the Global Campus platform:

Please visit your [GitHub Global Campus dashboard]("https://education.github.com/globalcampus/teacher"). You'll find the section **"Upgrade your academic organizations"** near the top of the page if you have organizations you can upgrade.

![Animation to Upgrade Organisation](https://user-images.githubusercontent.com/10368374/168666981-4e726973-3e0f-41c2-aec0-7c9181c41830.gif)

Click the **Upgrade to GitHub Team** button and then click the **Upgrade** button next to the organization name to immediately upgrade it for free to GitHub Team.

We hope you enjoy your GitHub Education benefits.
